pages/linux/sshuttle.md

```diff
@@ -0,0 +1,16 @@
+# sshuttle
+
+> Transparent proxy server that tunnels traffic over an SSH connection.
+> Doesn't require admin, or any special setup on the remote SSH server.
+
+- Forward all IPv4 TCP traffic via a remote SSH server:
+
+`sshuttle --remote={{username}}@{{sshserver}} {{0.0.0.0/0}}`
+
+- Forward all IPv4 TCP and DNS traffic via a remote SSH server:
+
+`sshuttle --dns --remote={{username}}@{{sshserver}} {{0.0.0.0/0}}`
+
+- Forward all IPv4 and IPv6 traffic via a remote SSH server:
```

Comment:
No need to mention "via a remote SSH server" after the 2nd time.

Reply:
Ok, sure.